algorithms/ar-lscsa/csharp/LargestSumContiguousSubArray.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+
+public class LargestSumContiguousSubArray
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6390379)